extension/alerts.js

```diff
@@ -135,12 +151,14 @@ const computeAlerts = async (url) => {
   const newAlerts = [];
   const domain = getDomain(url).toLowerCase();
   const visited = await visitedBeforeToday(domain);
+  const many_subdomains = hasManySubdomains(domain);
```

Comment:
nit: variable names should be lowerCamelCase

Reply:
Done